knapiks/mcchat.py

```diff
@@ -15,8 +15,18 @@
 import os
 
 twilio_client = Client(settings.TWILIO_ACCT_SID, settings.TWILIO_AUTH_TOKEN)
-admin_phone = Config.objects.get(mc_key='admin_phone').mc_value
-twilio_phone = Config.objects.get(mc_key='twilio_phone').mc_value
+
+mc_key = settings.MC_KEY
+admin_phone   = Config.objects.get(mc_key='admin_phone').mc_value
+twilio_phone  = Config.objects.get(mc_key='twilio_phone').mc_value
+ftp_host      = Config.objects.get(mc_key='ftp_host').mc_value
+ftp_login     = Config.objects.get(mc_key='ftp_login').mc_value
+ftp_pw_crypt  = Config.objects.get(mc_key='ftp_password').mc_value
+rcon_host     = Config.objects.get(mc_key='rcon_host').mc_value
+rcon_port     = Config.objects.get(mc_key='rcon_port').mc_value
+rcon_pw_crypt = Config.objects.get(mc_key='rcon_password').mc_value
+
+pending_messages = []
```

Comment:
You had multiple spots where configuration information was set. This is a script, so it makes sense to do all of that at once